### PR TITLE
Add tooltip text to Color Buttons

### PR DESCRIPTION
**Pull request by garciaolais**
This commit adds tooltip text to every color button in the tab color
picker.

### CODE REVIEW

**Comment by Chips1234** [comment]
If you need help, ask me. It took me a while to figure out the resource thing too.

File: src/cascadia/TerminalApp/ColorPickupFlyout.xaml [hunk 1]
```diff
@@ -24,82 +24,82 @@
                         <Setter Property="Margin" Value="2"/>
                     </Style>
                 </VariableSizedWrapGrid.Resources>
-                <Button Click="ColorButton_Click" AutomationProperties.Name="Crimson">
+                <Button Click="ColorButton_Click" AutomationProperties.Name="Crimson" ToolTipService.ToolTip="Crimson">
```
[the remark this answers is not in the document]
Comment (garciaolais) [reply src/cascadia/TerminalApp/ColorPickupFlyout.xaml]: Hi @Chips1234 

I'm confused  about your suggestion, Do I need to add a name like `ToolTipService.ToolTip=RS_(L"CrimsonTooltip")` or `x:Uid="CrimsonColorButton"` ?

Thanks

**Comment by Chips1234** [comment]
I believe it all works.

File: src/cascadia/TerminalApp/ColorPickupFlyout.xaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Chips1234) [reply src/cascadia/TerminalApp/ColorPickupFlyout.xaml]: @garciaolais I believe it all works.

[the remark this answers is not in the document]
Comment (zadjii-msft) [reply src/cascadia/TerminalApp/ColorPickupFlyout.xaml]: I'd be shocked if `ToolTipService.ToolTip=RS_(L"CrimsonTooltip")` compiled in XAML like that.

[the remark this answers is not in the document]
Comment (greg904) [reply src/cascadia/TerminalApp/ColorPickupFlyout.xaml]: This is weird. @Chips1234 I think you made the review on the old code, the commit 051d2fc already fixed the localization, no?

**Comment by Chips1234** [comment]
Oh sorry! I was confused. In XAML, you have to use “x:uid”

________________________________

File: src/cascadia/TerminalApp/Resources/en-US/Resources.resw [hunk 2]
```diff
@@ -347,4 +347,55 @@
...
+    <value>Crimson</value>
+  </data>
+  <data name="MediumSeaGreenColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Medium Sea Green</value>
+  </data>
+  <data name="SteelBlueColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Steel Blue</value>
+  </data>
+  <data name="MediumSeaGreenColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Medium Sea Green</value>
```
Comment (WSLUser) [inline src/cascadia/TerminalApp/Resources/en-US/Resources.resw:360]: This is a dupe of the one above

**Comment by DHowett** [comment]
Pushed a merge. I'll set this to automatically merge :)

**Comment by garciaolais** [comment]
> Thanks so much for doing this. Sorry for the slow review!

@DHowett  Thank you and thanks everyone! this made my day 😃

**Pull request merged**